src/toil/common.py

```diff
@@ -1143,12 +1148,12 @@ def import_file(self,
         return self._jobStore.import_file(src_uri, shared_file_name=shared_file_name, symlink=symlink)
 
     @deprecated(new_function_name='export_file')
-    def exportFile(self, jobStoreFileID: Union[FileID, str], dstUrl: str) -> None:
```

Comment:
Sometimes the file store methods need to be able to be called with strings, because sometimes user code will save a file ID to another file, as a string, and then load it again and present it to Toil. Cactus in particular does this.

@glennhickey Does Cactus still do that?